### PR TITLE
make stimulum less garbage

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl: 
balance: stimulum now makes you stun/sleep immune and some generic stamina heal 
/:cl:

it takes 30 seconds with a chem dispenser to make 100 u ephedrine (20 min worth of speed ) or a coin to get a nuka cola while stimulum takes effort and time and you get a shitty meme gas

cobble not welcome to post here

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1237,17 +1237,16 @@
...
+	L.remove_trait(TRAIT_SLEEPIMMUNE, id)
 	..()
 
-/datum/reagent/stimulum/on_mob_life(mob/living/carbon/M) // Has a speedup, and the anti-stun effects of nicotine.
-	M.AdjustStun(-20, 0)
-	M.AdjustKnockdown(-20, 0)
-	M.AdjustUnconscious(-20, 0)
-	M.adjustStaminaLoss(-0.5*REM, 0)
+/datum/reagent/stimulum/on_mob_life(mob/living/carbon/M) 
+	M.adjustStaminaLoss(-2*REM, 0)
```
Comment (kevinz000) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: UNDOCUMENTED CHANGE REEEEEEEEEE

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1237,17 +1237,16 @@
 
 /datum/reagent/stimulum/on_mob_add(mob/living/L)
 	..()
-	L.add_trait(TRAIT_GOTTAGOFAST, id)
+	L.add_trait(TRAIT_STUNIMMUNE, id)
```
Comment (kevinz000) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: may I suggest making it halve stuns instead of full immunity?

**Comment by ExcessiveUseOfCobblestone** [comment]
~~good change~~

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 2 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: scared?

**Comment by Tlaltecuhtli** [comment]
there are uhhhhhhhhh nanites, that shitty implant, surgery that already do half stun ( and dont stack) but they are like super easy and require nothing to make compared to stimulum

**Comment by XDTM** [comment]
fyi those do stack

**Comment by ExcessiveUseOfCobblestone** [comment]
@Therapist1 Ebola and BTFO them with nonharmful batons on harm intent :^)

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Tlaltecuhtli) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: not really much bcs stamina dmg is ass

Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: it needs to be documented regardless

**Comment by subject217** [comment]
`TRAIT_GOTTAGOREALLYFAST` would be a better choice than full stun immunity. The benefit is that you get methspeed without the small doses and constant brain damage.

**Comment by Kagus** [comment]
Not that anyone ever makes stimulum anyways, so it's kind of a moot point.

**Comment by ExcessiveUseOfCobblestone** [comment]
Because it's a pain to make, which is why stun immunity is okay in my book.

By using it though you make yourself de-facto valid since you can't be downed in any other form.

**Comment by Therapist1** [comment]
the real problem comes when you lace the stations air with it, imagine what happens when stubs and knockouts cease to exist

**Comment by ExcessiveUseOfCobblestone** [comment]
when do you have enough to toss into the air and it doesn't instantly dissipate?

**Comment by Whoneedspacee** [comment]
600 moles is about the most you could make with the best level of fusion taking you at least 30 minutes to achieve, maybe enough for a small room. It's definitely only useful in tanks really.

**Comment by Iamgoofball** [comment]
@subject217 what? no, the tradeoff for methspeed is the meth side effects

**Comment by subject217** [comment]
@Iamgoofball more balanced than full stun immunity for ingesting stimulum imo but eh in retrospect it's probably not

**Comment by cyclowns** [comment]
there's another issue that makes stimulum kind of ass and it has to do with breath cycles/reagent metabolization

with stim in its current state, even while paired with pluoxium in a tank with minmaxed ratios and everything, doesn't even make you fast 100% of the time because you breathe it too fast and the gas doesn't add enough into your system fast enough to compensate, so you get a weird spotty effect

i would think about maybe upping the amount of stimulum added per breath cycle or cranking how fast the reagent metabolizes down

**Review by Cyberboss — APPROVED** [review]
Subject to followup

**Pull request merged**